components/patient-data/indexing/pom.xml

```diff
@@ -136,5 +142,10 @@
       <version>${xwiki.version}</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>javax.servlet</groupId>
+      <artifactId>javax.servlet-api</artifactId>
+      <version>3.0.1</version>
```

Comment:
Don't specify the version, this is a managed dependency.

Also, it's only needed transitively during tests, so add `<scope>test</scope>`.